Utils/github_workflow_scripts/check_if_needs_to_fill_contribution_form.py

```diff
@@ -122,6 +122,30 @@ def arguments_handler():
     return parser.parse_args()
 
 
+def verify_labels(pr_label_names):
+    """
+    Verify that the external PR contains the following labels:
+
+    'Contribution Form Filled' and either one of 'Community'/'Partner'/'internal' labels
+
+    """
+    is_contribution_form_filled_label_exist = 'Contribution Form Filled' in pr_label_names
+    is_community_label_exist = 'Community' in pr_label_names
+    is_partner_label_exist = 'Partner' in pr_label_names
+    is_internal_label_exist = 'internal' in pr_label_names
+
+    if is_contribution_form_filled_label_exist:
+        if is_community_label_exist and not is_partner_label_exist and not is_internal_label_exist:
+            return True
+        elif not is_community_label_exist and is_partner_label_exist and not is_internal_label_exist:
+            return True
+        elif not is_community_label_exist and not is_partner_label_exist and is_internal_label_exist:
+            return True
+        else:
+            return False
```

Comment:
Instead of all this we can use python xor operator
```suggestion
        return is_community_label_exist ^ is_partner_label_exist ^ is_internal_label_exist
```

Reply:
nice, will change